docs/source/basics/customizing_interface.md

```diff
@@ -148,3 +109,292 @@ Each key in `metadata.parameters` must be the name of one of the parameters of t
 * `batch_table_column`: Whether this parameter should have a column to itself in the batch table at the top of the parameters page,
```

Comment:
I would add the type each takes, something like: e.x. `batch_table_column`: (Boolean) Whether...

Reply:
Also later we should add screenshots for most of how these correspond to the front end, i.e. display_name show the display name portion highlighted